src/bin/vip.js

```diff
@@ -29,7 +29,7 @@ const runCmd = async function () {
 		.command( 'app', 'List and modify your VIP applications' )
 		.command( 'backup', 'Generate a backup for VIP applications' )
 		.command( 'cache', 'Manage page cache for your VIP applications' )
-		.command( 'config', 'Set configuration for your VIP applications' )
+		.command( 'config', 'Manage environment configurations.' )
```

Comment:
This is a VERY minor nit, but it looks like the other descriptions don't end with a period (except for the `export` command), so it stood out to me.

Reply:
Well spotted, @brookewp ! We are updating the main command descriptions one at a time as we work through each command group. If you think it looks weird, we can just as easily update the main ones all at once.

Reply:
Ah, that makes sense! I think it's fine to leave it if the plan is to update them all to match eventually. 🙂